mocks/generate.go

```diff
@@ -16,3 +16,4 @@ package mocks
 
 //go:generate counterfeiter -o grafeas_client.go github.com/rode/rode/protodeps/grafeas/proto/v1beta1/grafeas_go_proto.GrafeasV1Beta1Client
 //go:generate counterfeiter -o grafeas_projects_client.go github.com/rode/rode/protodeps/grafeas/proto/v1beta1/project_go_proto.ProjectsClient
+//go:generate counterfeiter --fake-name FakeEsClient -o es_client.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/esutil.Client
```

Comment:
What are your thoughts on reusing the fake client from grafeas-elasticsearch (or more generally as we reuse code in different repositories)?

Then as we make changes to the client interface, we don't have to regenerate the fake and don't have to keep all of the versions of the fake client in Git history.

Reply:
I didn't even consider that, good idea.